<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slicky: The Do-Not-Disturb Light</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Status</string>
     </attribute>
     <widget class="QLabel" name="slackErrorLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>212</width>
        <height>112</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="statusButton1">
         <property name="text">
          <string>Set Status to Available</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="statusButton2">
         <property name="text">
          <string>Set Status to Busy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="statusButton3">
         <property name="text">
          <string>Set Status to Do Not Disturb</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>137</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="updateLightCheckBox">
         <property name="text">
          <string>Update Light Color</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="updateSlackCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update Slack</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Color Picker</string>
     </attribute>
     <widget class="QPushButton" name="colorPickerButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>181</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Select a Color</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="colorPreviewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="brightnessSlider">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="errorMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>611</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="connectToSlackButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>111</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Status:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Do Not Disturb</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCustomize">
    <property name="title">
     <string>Customize</string>
    </property>
    <addaction name="actionCustomize_Statuses"/>
    <addaction name="actionReconfigure_Slack"/>
   </widget>
   <addaction name="menuCustomize"/>
  </widget>
  <action name="actionCustomize_Statuses">
   <property name="text">
    <string>Customize Statuses</string>
   </property>
  </action>
  <action name="actionReconfigure_Slack">
   <property name="text">
    <string>Reconfigure Slack</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
